src/inmanta/deploy/state.py

```diff
@@ -170,9 +170,9 @@ class ResourceState:
 
     # FIXME: review / finalize resource state. Based on draft design in
     #   https://docs.google.com/presentation/d/1F3bFNy2BZtzZgAxQ3Vbvdw7BWI9dq0ty5c3EoLAtUUY/edit#slide=id.g292b508a90d_0_5
-    status: ComplianceStatus
-    deployment_result: DeploymentResult
-    blocked: BlockedStatus
+    compliance: Compliance
+    deploy_status: DeployStatus
```

Comment:
renames 3b & 5b. No comments needed, see respective threads for details.

Reply:
It is a bit weird that we have `compliance` (not `compliance_status`) and `deploy_status` not `deploy` 

I would go for 

`compliance`, `deploy_result` and `blocked`

Reply:
`deploy_result: DeployResult` or `deployment_result: DeploymentResult`? It was previously the latter.

Reply:
no strong preference